### PR TITLE
chore(RHTAPWATCH-241): Increase UWM prometheus retention duration and size

### PR DESCRIPTION
**Pull request by avi-biton**
- Set the retention time to 3d
- Set the retention size to 10GiB
- These settings would only apply to production and staging clusters 
- Update README file

### CODE REVIEW

File: components/monitoring/prometheus/base/uwm-config/uwm-config.yaml [hunk 1]
```diff
@@ -3,3 +3,8 @@ kind: ConfigMap
 metadata:
   name: user-workload-monitoring-config
   namespace: openshift-user-workload-monitoring
+data: 
+  config.yaml: | 
+    prometheus: 
+      retention: 3d 
+      retentionSize: 10GiB
```
Comment (yftacherzog) [inline components/monitoring/prometheus/base/uwm-config/uwm-config.yaml:10]: Should we exclude dev from this config line?

File: components/monitoring/prometheus/README.md [hunk 2]
```diff
@@ -10,8 +10,28 @@ Scrapes generic metrics produced by cAdvisor, kube-state-metrics, etc.
 Scrapes custom metrics provided by services deployed by the different teams.
 
 In Production and Staging, UWM Prometheus is enabled using OCM (since it maintains the
-Prometheus configurations).
-In Development it's enabled by deploying a configmap using ArgoCD.
+Prometheus configurations).  
+The retention is set to 3 days and the retention size is set to 10GiB.  
+It is defined in `components/monitoring/prometheus/base/uwm-config/uwm-config.yaml` and it is controlled by ArgoCD
```
Comment (yftacherzog) [inline components/monitoring/prometheus/README.md:15]:
```suggestion
It is defined in `components/monitoring/prometheus/base/uwm-config/uwm-config.yaml`
and it is controlled by ArgoCD
```

File: components/monitoring/prometheus/README.md [hunk 3]
```diff
@@ -10,8 +10,28 @@ Scrapes generic metrics produced by cAdvisor, kube-state-metrics, etc.
...
+
+
+In Development it's enabled without deploying a ConfigMap using ArgoCD (The ConfigMap is created automatically when UWM is enabled)  
+The retention is set to default (24h).
```
Comment (yftacherzog) [inline components/monitoring/prometheus/README.md:19]:
```suggestion
In Development it's enabled without deploying a ConfigMap using ArgoCD (The ConfigMap is
created automatically when UWM is enabled).

The retention is set to default (24h).  
```

**Review by yftacherzog — APPROVED** [review]
I suggest keeping the markdown lines short, but other than that. lgtm

**Review by amisstea — APPROVED** [review]

**Review by srivickynesh — APPROVED** [review]
/lgtm

**Pull request merged**